build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -123,4 +123,7 @@
   <Rules AnalyzerId="XmlDocumentationComments.Analyzers" RuleNamespace="XmlDocumentationComments.Analyzers">
     <Rule Id="RS0010" Action="Warning" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.NetCore.Analyzers" RuleNamespace="Microsoft.NetCore.Analyzers">
+    <Rule Id="CA9999" Action="None" /> <!-- We know the analyzers will fail during a bootstrap build -->
```

Comment:
:memo: I tried to conditionally disable this, but wasn't able to. We're also getting constrained by the non-standard way these rules are enabled. If we would just use a `<PackageReference>` to enable the analyzer packages, we could use different analyzer versions for the bootstrap build and everything would "just work".